test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-search-block-spec.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * External dependencies
+ */
+import { By } from 'selenium-webdriver';
+import config from 'config';
+import assert from 'assert';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../../lib/flows/login-flow.js';
+
+import GutenbergEditorComponent from '../../lib/gutenberg/gutenberg-editor-component';
+
+import * as driverManager from '../../lib/driver-manager';
+import * as driverHelper from '../../lib/driver-helper';
+import * as dataHelper from '../../lib/data-helper';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+const gutenbergUser =
+	process.env.COBLOCKS_EDGE === 'true' ? 'searchSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+describe( `[${ host }] Calypso Gutenberg Editor: Search Block (${ screenSize })`, function () {
+	this.timeout( mochaTimeOut );
+	let driver;
+
+	before( 'Start browser', async function () {
+		this.timeout( startBrowserTimeoutMS );
+		driver = await driverManager.startBrowser();
+	} );
+
+	describe( 'Insert Search Block and confirm it exists in te edit page: @parallel', function () {
+		it( 'Can log in', async function () {
+			this.loginFlow = new LoginFlow( driver, gutenbergUser );
+			return await this.loginFlow.loginAndStartNewPost( null, true );
+		} );
+
+		it( 'Can Insert the Search block', async function () {
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			await gEditorComponent.addBlock( 'Search' );
+			return await driverHelper.waitUntilElementLocatedAndVisible(
+				driver,
+				By.css( '.wp-block-search' )
+			);
+		} );
+
+		it( 'Can see search block elements', async function () {
+			const textLocator = By.css( '.wp-block-search__label' );
+			await driverHelper.waitUntilElementLocatedAndVisible( driver, textLocator );
+			await driverHelper.waitUntilElementLocatedAndVisible(
+				driver,
+				By.css( '.wp-block-search__input' )
+			);
+
+			return driverHelper.waitUntilElementLocatedAndVisible(
+				driver,
+				By.css( '.wp-block-search__button' )
+			);
+		} );
+
+		it( 'Can publish and view content', async function () {
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			// We need to save the post to get a stable post slug for the block's `url` attribute.
+			// See https://github.com/godaddy-wordpress/coblocks/issues/1663.
```

Comment:
This seems to be only for a specific block.  I don't think this comment applies here.